plugins/explore/src/components/GroupsExplorerContent/GroupsDiagram.tsx

```diff
@@ -222,21 +237,26 @@ export function GroupsDiagram() {
 
   return (
     <>
-      <DependencyGraph
-        nodes={nodes}
-        edges={edges}
-        nodeMargin={10}
-        direction={DependencyGraphTypes.Direction.RIGHT_LEFT}
-        renderNode={RenderNode}
-        className={classes.graph}
-      />
-      <Typography
-        variant="caption"
-        style={{ display: 'block', textAlign: 'right' }}
-      >
-        <ZoomOutMap style={{ verticalAlign: 'bottom' }} /> Use pinch &amp; zoom
-        to move around the diagram.
-      </Typography>
+      <Paper className={classes.graphWrapper}>
```

Comment:
Do you think the paper here is still needed? If worried it might just limit the size of the graph without any tangible benefit